modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,217 @@
++
+[NOTE]
+====
+This user will have Programmatic access enabled and the AdministratorAccess policy attached to it.
+====
++
+. Install and configure the AWS cli.
+.. Install the link:https://aws.amazon.com/cli/[aws-cli].
+.. Configure the `aws-cli` to use the AWS account that you would like to deploy your cluster into. Modify your `~/.aws/credentials` file to specify the correct `aws_access_key_id` and `aws_secret_access_key`.
++
+[source, bash]
+----
+$ cat ~/.aws/credentials
+
+[default]
+aws_access_key_id = <my-aws-access-key-id>
+aws_secret_access_key = <my-aws-secret-access-key>
+----
++
+.. Modify your `~/.aws/config` file to specify the AWS region you want to use.
++
+[source, bash]
+----
+$ cat ~/.aws/config
+
+[default]
+output = table
+region = us-east-2
+----
++
+.. To verify your configuration, run the following command to query the AWS api:
++
+[source, bash]
+----
+$ aws ec2 describe-regions
+
+---------------------------------------------------------------------------------
+|                                DescribeRegions                                |
++-------------------------------------------------------------------------------+
+||                                   Regions                                   ||
+|+-----------------------------------+-----------------------+-----------------+|
+||             Endpoint              |      OptInStatus      |   RegionName    ||
+|+-----------------------------------+-----------------------+-----------------+|
+||  ec2.eu-north-1.amazonaws.com     |  opt-in-not-required  |  eu-north-1     ||
+||  ec2.ap-south-1.amazonaws.com     |  opt-in-not-required  |  ap-south-1     ||
+||  ec2.eu-west-3.amazonaws.com      |  opt-in-not-required  |  eu-west-3      ||
+||  ec2.eu-west-2.amazonaws.com      |  opt-in-not-required  |  eu-west-2      ||
+||  ec2.eu-west-1.amazonaws.com      |  opt-in-not-required  |  eu-west-1      ||
+||  ec2.ap-northeast-2.amazonaws.com |  opt-in-not-required  |  ap-northeast-2 ||
+||  ec2.ap-northeast-1.amazonaws.com |  opt-in-not-required  |  ap-northeast-1 ||
+||  ec2.sa-east-1.amazonaws.com      |  opt-in-not-required  |  sa-east-1      ||
+||  ec2.ca-central-1.amazonaws.com   |  opt-in-not-required  |  ca-central-1   ||
+||  ec2.ap-southeast-1.amazonaws.com |  opt-in-not-required  |  ap-southeast-1 ||
+||  ec2.ap-southeast-2.amazonaws.com |  opt-in-not-required  |  ap-southeast-2 ||
+||  ec2.eu-central-1.amazonaws.com   |  opt-in-not-required  |  eu-central-1   ||
+||  ec2.us-east-1.amazonaws.com      |  opt-in-not-required  |  us-east-1      ||
+||  ec2.us-east-2.amazonaws.com      |  opt-in-not-required  |  us-east-2      ||
+||  ec2.us-west-1.amazonaws.com      |  opt-in-not-required  |  us-west-1      ||
+||  ec2.us-west-2.amazonaws.com      |  opt-in-not-required  |  us-west-2      ||
+|+-----------------------------------+-----------------------+-----------------+|
+----
++
+. Install `moactl`, the {product-title} command line utility.
```

Comment:
Elsewhere, you have command line utility constructed differently; You could standardize on:

> `moactl`, the Command-line Interface (CLI)

Or some such.